tests/keras_tests/feature_networks_tests/feature_networks/qat/qat_test.py

```diff
@@ -23,6 +23,9 @@
 import model_compression_toolkit as mct
 from model_compression_toolkit import quantizers_infrastructure as qi
 import os
+from model_compression_toolkit.exporter.model_wrapper.keras.builder.node_to_quantizer import QUANTIZATION_METHOD_2_ACTIVATION_QUANTIZER, QUANTIZATION_METHOD_2_WEIGHTS_QUANTIZER
```

Comment:
Add to tests: compare output of PTQ, QAT_init & QAT_finalize models. Same as in the PyTorch PR

Reply:
done